src/transactions/feed/queryHelper.ts

```diff
@@ -505,6 +508,122 @@ export const TRANSACTIONS_QUERY = gql`
     }
   }
 
+  fragment EarnDepositItemV3 on EarnDeposit {
```

Comment:
Do we need the V3 suffix for these?

Reply:
Fixed in 8dda925decf6f0a1e76868cc7463e000e9690ad5!